src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,36 @@
+package duke.tasks;
+/**
+ * 
+ * This represents an Event task with a time duration.
+ * 
+ * @author Rishabh Anand
+ * @version CS2103, AY21/22 Semester 1
+ * 
+ */
+
+import java.time.LocalDateTime;
+
+public class Event extends Task {
+    protected String timing;
+
+    public Event(String desc, String timing) {
+        super(desc);
+        this.timing = timing;
+    }
+
+    // returns string representation of the Event
+    @Override
+    public String toString() {
+        return "[E]" 
+                + super.toString()
+                + " (at: "
+                + this.timing
+                + ")";
```

Comment:
Consider condensing this into 2 lines at most. Slightly excessive line breaking.